nixos/modules/services/web-servers/nginx/default.nix

```diff
@@ -676,6 +677,14 @@ in
           Defines a group of servers to use as proxy target.
         '';
         default = {};
+        example = {
+          "backend_server" = {
+            servers = { "127.0.0.1:8000" = {}; };
+            extraConfig = ''
+              keepalive 16;
+            '';
+          };
+        };
```

Comment:
```suggestion
        literalExample = ''
          "backend_server" = {
            servers = { "127.0.0.1:8000" = {}; };
            extraConfig = ''
              keepalive 16;
            '';
          };
        ''
```
like this?

Reply:
`example = literalExample '' ... '';` IIRC :)